codewars8.py

```diff
@@ -0,0 +1,36 @@
+# https://www.codewars.com/kata/58b3c2bd917a5caec0000017/train/python
+# works locally, but codewars returns error:/
+# not all arguments converted during string formatting (line 17 I think)
+
+def sum_groups(arr):
+    arr.append("hehe")
+    batman = True
+    while batman:
```

Comment:
I think in general you can skip declaring `batman` variable and just use `while True:`. Break the loop when it should be stopped.